packages/labs/gen-wrapper-vue/src/lib/vite.config-template.ts

```diff
@@ -35,14 +36,22 @@ export default {
       external: (id: string) => !id.match(/^((\\w:)|(\\.?[\\\\/]))/),
       input: [
         ${Object.keys(sfcFiles)
-          .map((path) => `'./${path}'`)
+          .map((path) => `'./${path.replace(/\\/g, '/')}'`)
           .join(', ')}
       ],
       preserveModules: false,
       preserveEntrySignatures: true,
       output: {
         format: 'es',
-        entryFileNames: ({ name }) => \`\${name}.js\`,
+        // For subpath component, we should output it in subpath.
+        // Otherwise, if there are files with the same file name in the subPath and rootPath,
+        // output to the same directory will conflict.
+        entryFileNames: ({ name, facadeModuleId }) => {
+          const sourceFileDir = path.dirname(facadeModuleId);
+          const relativePath = path.relative(__dirname, sourceFileDir);
+          const targetName = path.join(relativePath, \`\${name}.js\`).replace(/\\\\/g, '/').replace(/^src\\\//, '');
+          return targetName;
+        },
```

Comment:
I'm trying to think over this logic. I think we might be able to do `preserveModules: true` and `preserveModulesRoot: 'src'` to avoid this. https://rollupjs.org/configuration-options/#output-preservemodulesroot

Reply:
I have used `preserveModules` to replace these code and it does work.